app/models/intake/ctc_intake.rb

```diff
@@ -287,13 +298,6 @@ class Intake::CtcIntake < Intake
   attribute :primary_prior_year_agi_amount, :money
   attribute :spouse_prior_year_agi_amount, :money
 
-  attr_encrypted :primary_ssn, key: ->(_) { EnvironmentCredentials.dig(:db_encryption_key) }
```

Comment:
i moved these onto the base intake because it makes the data migration simpler. will be removed entirely soon.